soh/CMake/Linux32bit-toolchain.cmake

```diff
@@ -0,0 +1,18 @@
+# the name of the target operating system
+set(CMAKE_SYSTEM_NAME Linux)
+
+# which compilers to use for C and C++
+set(CMAKE_C_COMPILER gcc)
```

Comment:
One useful thing we recently did in the Makefiles was being able to set a custom compiler via `CC` and `CXX` flags. Example: `CC=gcc-12 CXX=g++-12`. Is that possible here?

Reply:
The idea is to use adjusted toolchain files, e.g. gnu-12-toolchain.cmake or clang-14-toolchain.cmake